README.md

```diff
@@ -27,11 +27,8 @@ or
 - IOS
 
 To install all Pods run the command in root direcroty:
-`cd ios`
 
-further
-
-`pod install`
+`yarn pods`
```

Comment:
The rest of the README only uses yarn, so I assumed it would be better to only include the yarn commands? Can change this if you believe its best to add both though!

Reply:
No it's okay then, cheers